### DISCUSSION TITLE
24/02/01

### DISCUSSION THREAD

**Discussion by 00TaciTa00** [codeit-fe2-2/taskify#48] [Daily Scrum]
- 일단 이서영은 Table 만들거에요
- PR 리뷰 및 머지 방식에 대하여
  - 기존 : 모두가 approve 하면 머지하기
  - 현재 룰 : 1명 이상 approve 시 머지 가능
 - 규칙
   - PR 올리면 무조건 PR 올렸다고 디스코드에 말하기
   - 꼭 올릴 때 `문제 없으면 ~에는 머지할게요` <- 올리는 사람이 유동적으로 정하기
   - 머지하는 건 PR 올린 당사자가
   - 수정사항 생기면?
     - 원래 하던 일이 무엇이든 간에 수정 사항 먼저 작업하기
     - 본래 머지하기로 한 시간 내에 작업하는 걸 목표로 하되 문제가 생기면 디코에 올리기
- 무슨 일 생기면 무조건!!!!! 디스코드에!!!! 말해주세요!!!!!

**Comment by ckswnskfk** [comment]
- 나의 대시보드 페이지
- 모달 context
